metricbeat/module/docker/network/data.go

```diff
@@ -5,32 +5,70 @@ import (
 	"github.com/elastic/beats/metricbeat/mb"
 )
 
-func eventsMapping(netsStatsList []NetStats) []common.MapStr {
-	myEvents := []common.MapStr{}
+func eventsMapping(r mb.ReporterV2, netsStatsList []NetStats) {
 	for _, netsStats := range netsStatsList {
-		myEvents = append(myEvents, eventMapping(&netsStats))
+		eventMapping(r, &netsStats)
 	}
-	return myEvents
 }
 
-func eventMapping(stats *NetStats) common.MapStr {
-	event := common.MapStr{
-		mb.ModuleDataKey: common.MapStr{
-			"container": stats.Container.ToMapStr(),
+func eventMapping(r mb.ReporterV2, stats *NetStats) {
+	container := stats.Container.ToMapStr()
+	container.Put("runtime", "docker")
+
+	// Inbound traffic event
+	r.Event(mb.Event{
```

Comment:
I would still expect that 1 event is sent out with all the stats inside. It seems now we send out 3 events? Why not combine inbound and outbound?

Reply:
I did that initially, but then I saw that we have a `network.direction` field in ECS, then I though that according to ECS we'd have different events for inbound and outbound traffic. Not sure what would be the use case for this field if not used for something like this.